e2e/test-component/scenarios/embedding-sdk/interactive-question.cy.spec.tsx

```diff
@@ -0,0 +1,281 @@
+  beforeEach(() => {
+    signInAsAdminAndEnableEmbeddingSdkForComponentTests();
+
+    createQuestion(
+      {
+        name: "47563",
+        query: {
+          "source-table": ORDERS_ID,
+          aggregation: [["max", ["field", ORDERS.QUANTITY, null]]],
+          breakout: [["field", ORDERS.PRODUCT_ID, null]],
+          limit: 2,
+        },
+      },
+      { wrapId: true },
+    );
+
+    cy.signOut();
+
+    sdkJwtSignIn();
+  });
+
+  it("should show question content", () => {
+    mountInteractiveQuestion();
+
+    getSdkRoot().within(() => {
+      cy.findByText("Product ID").should("be.visible");
+      cy.findByText("Max of Quantity").should("be.visible");
+    });
+  });
+
+  it("should not fail on aggregated question drill", () => {
+    mountInteractiveQuestion();
+
+    cy.wait("@cardQuery").then(({ response }) => {
+      expect(response?.statusCode).to.equal(202);
+    });
+
+    cy.findAllByTestId("cell-data").last().click();
+
+    cy.on("uncaught:exception", error => {
+      expect(
+        error.message.includes(
+          "Error converting :aggregation reference: no aggregation at index 0",
+        ),
+      ).to.be.false;
+    });
+
+    popover().findByText("See these Orders").click();
+
+    cy.icon("warning").should("not.exist");
+  });
+
+  it("should be able to hide columns from a table", () => {
+    mountInteractiveQuestion();
+
+    cy.wait("@cardQuery").then(({ response }) => {
+      expect(response?.statusCode).to.equal(202);
+    });
+
+    tableInteractive().findByText("Max of Quantity").should("be.visible");
+
+    tableHeaderClick("Max of Quantity");
+
+    popover()
+      .findByTestId("click-actions-sort-control-formatting-hide")
+      .click();
+
+    tableInteractive().findByText("Max of Quantity").should("not.exist");
+  });
+
+  it("can save a question to a default collection", () => {
+    mountInteractiveQuestion();
+
+    saveInteractiveQuestionAsNewQuestion({
+      entityName: "Orders",
+      questionName: "Sample Orders 1",
+    });
+
+    cy.wait("@createCard").then(({ response }) => {
+      expect(response?.statusCode).to.equal(200);
+      expect(response?.body.name).to.equal("Sample Orders 1");
+      expect(response?.body.collection_id).to.equal(null);
+    });
+  });
+
+  it("can save a question to a selected collection", () => {
+    mountInteractiveQuestion();
+
+    saveInteractiveQuestionAsNewQuestion({
+      entityName: "Orders",
+      questionName: "Sample Orders 2",
+      collectionPickerPath: ["Our analytics", "First collection"],
+    });
+
+    cy.wait("@createCard").then(({ response }) => {
+      expect(response?.statusCode).to.equal(200);
+      expect(response?.body.name).to.equal("Sample Orders 2");
+      expect(response?.body.collection_id).to.equal(FIRST_COLLECTION_ID);
+    });
+  });
+
+  it("can save a question to a pre-defined collection", () => {
+    cy.get("@questionId").then(questionId => {
+      mountSdkContent(
```

Comment:
what do you think about making `mountInteractiveQuestion` accept props instead of having to call `mountSdkContent` manually like this?